chsdi/models/vector/bafu.py

```diff
@@ -2390,3 +2390,25 @@ class LandesforstinventarSchutzwaldregion(Base, LandesforstinventarRegion, Vecto
     __bodId__ = 'ch.bafu.landesforstinventar-schutzwaldregionen'
 
 register(LandesforstinventarSchutzwaldregion.__bodId__, LandesforstinventarSchutzwaldregion)
+
+
+class AmphibienwanderungVerkehrskonflikte(Base, Vector):
+    __tablename__ = 'amphibienwanderung_verkehrskonflikte'
+    __table_args__ = ({'schema': 'fauna', 'autoload': False})
+    __bodId__ = 'ch.bafu.amphibienwanderung-verkehrskonflikte'
+    __template__ = 'templates/htmlpopup/amphibienwanderung-verkehrskonflikte.mako'
+    __queryable_attributes__ = ['nr_datenbank', 'name']
+    __label__ = 'name'
+    id = Column('nr_datenbank', Integer, primary_key=True)
```

Comment:
is this attribute delivered with the data and supposed to be stable?

Reply:
yes, this is supposed to be stable